tools/enso4igv/src/main/java/org/enso/tools/enso4igv/EnsoSbtProject.java

```diff
@@ -0,0 +1,56 @@
+package org.enso.tools.enso4igv;
+
+import java.io.IOException;
+import org.netbeans.api.project.Project;
+import org.netbeans.spi.project.ProjectFactory;
+import org.netbeans.spi.project.ProjectState;
+import org.openide.filesystems.FileObject;
+import org.openide.util.Lookup;
+import org.openide.util.lookup.Lookups;
+import org.openide.util.lookup.ServiceProvider;
+
+public class EnsoSbtProject implements Project {
+    private final FileObject prj;
+    private final ProjectState ps;
+    private final Lookup lkp;
+
+    private EnsoSbtProject(FileObject fo, ProjectState ps) {
+        this.prj = fo;
+        this.ps = ps;
+        this.lkp = Lookups.fixed(
+            this,
+            new EnsoSbtClassPathProvider(this)
+        );
+    }
+
+    public FileObject getProjectDirectory() {
+        return prj;
+    }
+
+    public Lookup getLookup() {
+        return lkp;
+    }
+
+    @Override
+    public String toString() {
+        return "EnsoSbtProject{" + "prj=" + prj + '}';
+    }
+
+    @ServiceProvider(service = ProjectFactory.class)
+    public static final class Factory implements ProjectFactory {
+        public boolean isProject(FileObject fo) {
+            return fo.getFileObject(".enso-sources") != null;
```

Comment:
Can you define a constant in one place at least for `enso4igv` project?

Reply:
Done in [3de9908](https://github.com/enso-org/enso/pull/3810/commits/3de99088f149fa8f6b64e5426db4d80c0278e21c)